doc/api/http.md

```diff
@@ -189,6 +189,9 @@ This method can be overridden by a particular `Agent` subclass. If this
 method returns a falsy value, the socket will be destroyed instead of persisting
 it for use with the next request.
 
+`socket` argument can be an instance of {stream.Duplex}, a superclass of
```

Comment:
Nit: I would either write "`socket` can be an instance..." or "The `socket` argument can be an instance...".